ray-operator/config/samples/ray_v1alpha1_rayjob.shutdown.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: ray.io/v1
+apiVersion: ray.io/v1alpha1
```

Comment:
@kevin85421 I wasn't sure if it was safe to delete the v1alpha1 manifests, so I kept it and just referenced v1alpha1 for now.

Reply:
You can directly rename the file, as we no longer require any YAML manifests with `apiVersion: ray.io/v1alpha1`. Moreover, our naming convention is `ray-{cluster | job | service}.{something_important_for_this_yaml}.yaml`. In this case, we can rename it to `ray-job.shutdown.yaml`.

Some YAMLs don't follow the naming style for their filenames. We can open another PR to improve that.

Reply:
okay, thanks. Will update to follow this convention

Reply:
updated to directly rename the files